Controller/MediaAdminController.php

```diff
@@ -42,22 +41,20 @@ public function createAction(Request $request = null)
     /**
      * {@inheritdoc}
      */
-    public function render($view, array $parameters = array(), Response $response = null, Request $request = null)
+    public function render($view, array $parameters = array(), Response $response = null)
```

Comment:
Yes, he removes a paramater, but this isn't BC. Adding a parameter with a default value is, but not the other way round.